src/components/Breadcrumb.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <w-breadcrumbs :items="breadcrumbData" xl />
+</template>
+
+<script setup>
+import { ref, watch } from "vue";
+import { useRoute } from "vue-router";
+
+const route = useRoute()
+const breadcrumbData = ref([]);
+
+const getBreadcrumbData = () => {
+  let matched_list = route.matched;
+  breadcrumbData.value = create_wbreadcrumb_list(matched_list)
+};
+
+watch(
+  route,
+  () => {
+    getBreadcrumbData();
+  },
+  {
+    immediate: true,
+  }
+);
+
+function isHome(item) {
+  return item.name === "home";
+};
+
+function create_wbreadcrumb_list(matched_list) {
```

Comment:
I don't think we need to call this `create_wbreadcrumb_list` just because of the widget set we are using simply `create_breadcrumb_list` is sufficient.